numpyro/util.py

```diff
@@ -231,26 +248,20 @@ def _update_progress_bar(iter_num):
 
         _ = lax.cond(
             iter_num == 1,
-            lambda _: host_callback.id_tap(
-                _update_tqdm, 0, result=iter_num, tap_with_device=True
-            ),
-            lambda _: iter_num,
+            lambda _: io_callback(_update_tqdm, None, -1, 0),
```

Comment:
i'm not sure why -1, 0 is used here

Reply:
The -1 was a mistake, changed to 0,0.   The second 0 is consistent with the current implementation.  I believe this is 0 instead on 1 so that subsequent calls using print_rate and remainder add to the total.  But it does seem to be off by 1